.makerc

```diff
@@ -0,0 +1,9 @@
+OPENWISP_CONTROLLER_SOURCE:=https://github.com/openwisp/openwisp-controller/archive/0.4.0.tar.gz
```

Comment:
I think this kind of file should not be present in the original repo and should be added to `.gitignore`. It should be included in the `Makefile` only if present in the filesystem. 

The default values in the `ARG` statements should be used in most cases.

The user who needs to customize the source URLs will have to create this file and define their customization here.

Reply:
I think templating `.makerc` with default value is okay though. With this template way, we shouldn't explain the values that can be changed in the docs, and I think saving some users time writing those variables... What do you think?

Reply:
@marfgold1 can @nemesisdesign 's suggestion be done? :smile:
I think it's an easy change.

Reply:
@marfgold1 I don't like the approach you suggest for one reason:

if users of this project change their values, it will show up as a change on git.
Hence a change in configuration shows up as a change in the code of the project.
As a general rule, configuration should be kept separate from committed code in order to not confuse the two.